using_openshift/console.adoc

```diff
@@ -10,25 +10,25 @@
 toc::[]
 
 == Overview
-With the OpenShift Management Console you can visualize and browse the contents of your projects.
+With the OpenShift Management Console, you can visualize and browse the contents of your projects.
 
-NOTE: JavaScript must be enabled to use the Management Console. For the best experience use a web browser that supports link:http://caniuse.com/#feat=websockets[websockets].
+NOTE: JavaScript must be enabled to use the Management Console. For the best experience, use a web browser that supports link:http://caniuse.com/#feat=websockets[websockets].
 
 === Navigating
```

Comment:
From here on, all of the headings ("Navigating", "Filtering by labels", and "Creating from template") can be second-level headers (==) like "Overview", instead of third-level. Also the latter two can be title case.